docs/index.rst

```diff
@@ -368,17 +368,17 @@ Wrappers
     1.1. For earlier versions, see the "old-style hook wrappers" section below.
     The two styles are fully interoperable.
```

Comment:
We should mention the distinct option names next to each other, as this will help users search for them and be told of the difference explicitly.

```suggestion
    New-style hooks wrappers are declared with ``wrapper=True``, while old-style hook wrappers are declared with ``hookwrapper=True`` (for backward compatibility).

    The two styles are fully interoperable between plugins using different styles. However within the same plugin we recommend using only one style, for consistency.
```

Reply:
Thanks! I'll apply this, except for the "(for backward compatibility)" part, I think it is potentially confusing.

Reply:
Ahh right, might give the impression we are deprecating old-style hook wrappers, which is not the case.